workers/unity/Packages/io.improbable.gdk.core/Dynamic/ComponentDatabase.cs

```diff
@@ -43,6 +43,16 @@ public static IComponentMetaclass GetMetaclass(uint componentId)
             return metaclass;
         }
 
+        public static IComponentMetaclass GetMetaclass<T>() where T : ISpatialComponentData
```

Comment:
This was added to avoid having to get the component id first.
However mostly to make sonar stop complaining about the generic statics in Reader.cs

Reply:
Ah that explains https://github.com/spatialos/gdk-for-unity/pull/1364#discussion_r465687105 then 😂